core/channels/setup_structlog.py

```diff
@@ -9,6 +9,8 @@
 from pythonjsonlogger import jsonlogger
 from structlog.contextvars import merge_contextvars
 
+import sanic
```

Comment:
We should investigate this, our import formatting with isort is different between core and action-server.

Reply:
Fixed, this was a problem related to mypy and third party libraries